scripts/python/replay_session.py

```diff
@@ -1,29 +1,28 @@
 """Script that will replay sessions and allow us to simulate new model predictions on that data."""
 import json
 import logging as logger
-from typing import Tuple
-from pathlib import Path
 import pickle
+from pathlib import Path
+from typing import Tuple
 
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 import seaborn as sns
-from bcipy.config import (
-    RAW_DATA_FILENAME,
-    TRIGGER_FILENAME,
-    DEFAULT_PARAMETER_FILENAME, SESSION_DATA_FILENAME,
-    DEFAULT_DEVICE_SPEC_FILENAME,
-)
-from bcipy.helpers.acquisition import analysis_channels
+
```

Comment:
I like that we can still use this script, but I do worry about its long-term survival. Could we add a replay feature to the simulator? It would permit slightly different questions and add a sampling strategy. The visualization was also beneficial (and the most used to communicate outwardly) and, I think, could be used more generally in the simulator output.   

I'd be okay with deprecating this script (and most others) before the 2.0 release, even if it can't be rolled in.

Reply:
I didn't actually make any changes to this script, but saving it organized the imports in my editor. Replicating this behavior should be fairly easy to do in the simulator. I think it's just a matter of having a stateful sampler.